### PR TITLE
[TACHYON-1543] Update - Show permission info in UI and CLI

### PR DESCRIPTION
**Pull request by sundapeng**
Jira: https://tachyon.atlassian.net/browse/TACHYON-1543

The pull request is to show permission info in UI and CLI

@apc999 , @jsimsa , @yupeng9 , @dongche

 Thanks!

### CODE REVIEW

**Comment by sundapeng** [comment]
@apc999 , @jsimsa , @yupeng9 , @dongche, I will rebase the pr once https://github.com/amplab/tachyon/pull/2394 is merged. Wait a moment for https://github.com/amplab/tachyon/pull/2394 is also okay.

**Comment by yupeng9** [comment]
Thanks. #2394 is merged. Let us know after you have rebased.

**Comment by sundapeng** [comment]
Rebased with branch  `authorization`

File: common/src/main/java/tachyon/Constants.java [hunk 1]
```diff
@@ -35,6 +35,17 @@
...
   public static final String ANSI_WHITE = "\u001B[37m";
 
+  public static final String LS_FORMAT_FILE_SIZE = "%-10s";
+  public static final String LS_FORMAT_CREATE_TIME = "%-25s";
+  public static final String LS_FORMAT_FILE_TYPE = "%-15s";
+  public static final String LS_FORMAT_PERMISSION = "%-15s";
+  public static final String LS_FORMAT_USER_NAME = "%-15s";
+  public static final String LS_FORMAT_GROUP_NAME = "%-15s";
+  public static final String LS_FORMAT_FILE_PATH = "%-5s";
+  public static final String COMMAND_FORMAT_LS = LS_FORMAT_FILE_SIZE + LS_FORMAT_CREATE_TIME
```
Comment (yupeng9) [inline common/src/main/java/tachyon/Constants.java]: It seems this change conflicts with the one in https://github.com/amplab/tachyon/pull/2406/files ?

Comment (sundapeng) [inline common/src/main/java/tachyon/Constants.java]: Yes, it seems https://github.com/amplab/tachyon/pull/2406 will fix `ls`/`lsr` command at master, But web page also need to be fixed. do you think I should remove the `ls` part of the pull request? if yes, should I move it to master?

Comment (yupeng9) [inline common/src/main/java/tachyon/Constants.java]: Yeah, and also you may want to merge master into your authorization branch.

**Comment by apc999** [comment]
@sundapeng You may remove tfsshell part (ls) and leave the fix to web in this PR. Otherwise it looks good to me. Thanks for the patch!

**Comment by sundapeng** [comment]
Thank @apc999 and @yupeng9 for your review. I found creating a pr for master will be better than branch `authorization`, I created https://github.com/amplab/tachyon/pull/2502 for master and leave the fix to web in that PR.

**Pull request closed without merging**